sale_product_multi_add/wizards/sale_import_products_view.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    
+    <record id="view_import_product_to_sale" model="ir.ui.view">
+        <field name="name">import.products.to.sale.form</field>
+        <field name="model">sale.import.products</field>
+        <field name="arch" type="xml">
+        <form string="Products">
+        <field name="products" />
+        <footer>
```

Comment:
It will be nice if the indentation can be fixed in this xml file.

Reply:
Thanks for code review :smiley: